HARK/distribution.py

```diff
@@ -1404,3 +1579,11 @@ def sample(s):
         array_sample = np.frompyfunc(sample, 1, 1)
 
         return array_sample(state)
+
+
+def Expected(func=None, dist=None, args=(), labels=False):
```

Comment:
This is a function with a capitalized name.
That breaks the PyPI naming standard.

Could it be `expected`?

It could potentially be calleable Python class but that would be doing a lot of gymnastics for an idiosyncratic stylizistic win.

Reply:
We would *really* like to have the Expectation operator capitalized, as in written math. `sympy` does create a class for it, but you are right that this is a lot for what is just a function in our case

https://github.com/sympy/sympy/blob/77f1d79c705da5e9b3dee456a14b1b4e92dd620c/sympy/stats/symbolic_probability.py#L134-L323

Reply:
@sbenthall Here I went with args as a tuple because I couldn't figure out how to keep args and label separate.